lib/hardware/memories/ram/iob_ram_tdp/iob_ram_tdp.py

```diff
@@ -3,7 +3,166 @@ def setup(py_params_dict):
         "original_name": "iob_ram_tdp",
         "name": "iob_ram_tdp",
         "version": "0.1",
-        "generate_hw": False,
+        "confs": [
+            {
+                "name": "HEXFILE",
+                "type": "P",
+                "val": '"none"',
+                "min": "NA",
+                "max": "NA",
+                "descr": "Name of file to load into RAM",
+            },
+            {
+                "name": "ADDR_W",
+                "type": "P",
+                "val": "6",
+                "min": "0",
+                "max": "NA",
+                "descr": "Address bus width",
+            },
+            {
+                "name": "DATA_W",
+                "type": "P",
+                "val": "8",
+                "min": "0",
+                "max": "NA",
+                "descr": "Data bus width",
+            },
+            {
+                "name": "MEM_NO_READ_ON_WRITE",
+                "type": "P",
+                "val": "1",
+                "min": "0",
+                "max": "NA",
+                "descr": "",
+            },
+            {
+                "name": "mem_init_file_int",
```

Comment:
```suggestion
                "name": "MEM_INIT_FILE_INT",
```
Parameter should be uppercase like in the snippet:
https://github.com/IObundle/iob-soc/pull/943/files/a27903b036def55f4141eee63eb10946e0612ab9..f1f2f4c5bf0214a9e65f172ca6d62f7e9d273c46#diff-8df87ad132a2f20b1f1965c148aa90f8201fa4e03efa13d08393bf5c331b07d0R138